.github/workflows/build.yml

```diff
@@ -330,6 +370,8 @@ jobs:
         run: ./admin/coverage --lcov --output-path final.info
 
       - name: Report to codecov.io
```

Comment:
add comment like this before `name:` line:

    # TODO: CONFIGURE & ENABLE CODECOV FOR THIS FORK